extensions/wikia/PhalanxII/PhalanxSpecialController.class.php

```diff
@@ -314,15 +314,15 @@ public function unblock() {
 		$token = $this->request->getVal( 'token' );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "token" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)